/**
 * Created with IntelliJ IDEA.
 * User: glukki
 * Date: 01.05.12
 * Time: 21:42
 * To change this template use File | Settings | File Templates.
 */

import "Backpack.proto";
import "Flowerbed.proto";
option java_package = "com.kloomba.app.protobuf";
option java_outer_classname = "FlowerbedAddProto";

//request: /flowerbed/add?lat=59955835&lon=30328488
message FlowerbedAdd {
    optional int32 timestamp = 1;
    optional Flowerbed flowerbed = 2; //new flowerbed
    optional Backpack backpack = 3; //updated backpack
}